dashboard/src/utils/axiosInstance.js

```diff
@@ -0,0 +1,41 @@
+import axios from "axios";
+import { LOCAL_BASE_URL } from "./apiConstants";
+import Cookies from "js-cookie";
+import { showSessionExpired } from "actions/toastActions";
+import store from "store/store";
+
+const { dispatch } = store;
+
+let token = Cookies.get("token") ? Cookies.get("token") : null;
+
+const axiosInstance = axios.create({
+  baseURL: LOCAL_BASE_URL,
+  responseType: "json",
+  headers: {
+    Authorization: `Bearer ${token}`,
+  },
+});
+
+axiosInstance.interceptors.request.use(async (req) => {
+  token = Cookies.get("token") ? Cookies.get("token") : null;
+  req.headers.Authorization = `Bearer ${token}`;
+
+  return req;
+});
+/** Intercept any unauthorized request.
+ * dispatch logout action accordingly **/
+const UNAUTHORIZED = 401;
```

Comment:
It seems weird that we would roll our own definition for this...isn't there a standard one that we could import from somewhere? (I found [http-status-codes](https://www.npmjs.com/package/http-status-codes)...I have no idea whether that's a good source or not....)